examples/add.code

```diff
@@ -1,5 +1,5 @@
 // only using add, no need to flatten
-def main():
+def main(a):  // or def main(private a):
```

Comment:
Thanks for the contribution! I think you can remove that comment, this example is fine without `private`

Reply:
Done. Thanks for this project :+1:

Reply:
Merged! Thank you!